README.md

```diff
@@ -153,7 +153,30 @@ let options = {
   frameLength: 512,
   outputSampleRate: 16000,
   deviceId: null,
-  filterOrder: 50
+  filterOrder: 50,
+};
+
+WebVoiceProcessor.setOptions(options);
+```
+
+### Custom Recorder Processor
+
+**NOTE**: We won't be responsible for any bugs related to using custom-made recorder processor.
```

Comment:
move this below the first paragraph that explains what a custom recorder processor does
```suggestion
**NOTE**: Issues related to custom recorder processor implementations are out of the scope of this repo.
```